oak-run/src/main/java/org/apache/jackrabbit/oak/run/RevisionsCommand.java

```diff
@@ -146,6 +147,10 @@ private static class RevisionsOptions extends Utils.NodeStoreOptions {
             delay = parser
                     .accepts("delay", "introduce delays to reduce impact on system").withRequiredArg()
                     .ofType(Double.class).defaultsTo(0.0);
+            fullGcDelayFactor = parser
+                    .accepts("fullGcDelayFactor", "introduce delays after each fullGc batch to reduce load " +
+                            "on system")
+                    .withRequiredArg().ofType(Double.class).defaultsTo(0.0);
```

Comment:
as this will now be a command line parameter, we could use a more conservative value here. this would protect against unknowingly overloading a production system.

Reply:
let's use the same value as in OSGi configurations.

Reply:
addressed in https://github.com/apache/jackrabbit-oak/pull/1814/commits/8cb54bf67b1a9ffa24e5017fb43322887f4aa287